_source/atoms/headline/Headline.scss

```diff
@@ -11,6 +11,10 @@
   margin-top: 2rem;
   padding-bottom: 1rem;
 
+  &.headline--dark-mode {
+    color: #fff;
```

Comment:
You usually use variables for colors, does it make sense to stick to one design "system"?

Reply:
Will do 👍